File: app/src/main/java/com/subodh/location_demo_android/MyViewHolder.java
package com.subodh.location_demo_android;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class MyViewHolder extends RecyclerView.ViewHolder {
    TextView tvLatLng;

    public MyViewHolder(@NonNull View itemView) {
        super(itemView);
        tvLatLng = itemView.findViewById(R.id.tvLatLng);
    }
}

File: app/src/main/java/com/subodh/location_demo_android/MyViewHolder.kt
package com.subodh.location_demo_android

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

class MyViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var tvLatLng: TextView = itemView.findViewById(R.id.tvLatLng)

}